Assets/Resources/Materials/M_GuardProto-0.mat

```diff
@@ -0,0 +1,83 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!21 &2100000
+Material:
+  serializedVersion: 8
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_Name: M_GuardProto-0
```

Comment:
We don't name materials or anything else with a dash in the middle. It should be M_GuardProto_0. And since there won't be multiple prototype guards, no need to add a _0 at all. Rename M_GuardProto-0 to M_Guard